app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,284 @@
+                app:layout_constraintEnd_toStartOf="@id/second_image_view"
+                app:layout_constraintTop_toBottomOf="@id/cast_Text_View" />
+
+            <ImageView
+                android:id="@+id/second_image_view"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:src="@drawable/img_two"
+                app:layout_constraintDimensionRatio="1:1"
+                app:layout_constraintHorizontal_weight="1"
+                android:layout_marginStart="8dp"
+                android:layout_marginEnd="8dp"
+                app:layout_constraintHorizontal_chainStyle="spread_inside"
+                app:layout_constraintStart_toEndOf="@id/first_actor_image_view"
+                app:layout_constraintEnd_toStartOf="@id/third_image_view"
+                app:layout_constraintTop_toBottomOf="@id/cast_Text_View" />
+            <ImageView
+                android:id="@+id/third_image_view"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:src="@drawable/img_three"
+                app:layout_constraintHorizontal_weight="1"
+                app:layout_constraintDimensionRatio="1:1"
+                android:layout_marginEnd="8dp"
+                app:layout_constraintStart_toEndOf="@id/second_image_view"
+                app:layout_constraintTop_toBottomOf="@id/cast_Text_View"
+                app:layout_constraintEnd_toStartOf="@id/fourth_image_view" />
+            <ImageView
+                android:id="@+id/fourth_image_view"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                app:layout_constraintHorizontal_weight="1"
+                app:layout_constraintDimensionRatio="1:1"
+                android:scaleType="centerCrop"
+                android:src="@drawable/img_fore"
+                app:layout_constraintStart_toEndOf="@id/third_image_view"
+                app:layout_constraintTop_toBottomOf="@id/cast_Text_View"
+                app:layout_constraintEnd_toEndOf="@id/cast_Text_View"/>
+            <TextView
+                android:id="@+id/info_One"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="6dp"
+                android:fontFamily="@font/robotoblack"
+                android:text="@string/one_name_act"
+                android:textColor="@color/actor_name_color"
+                android:textSize="12dp"
+                android:textStyle="bold"
+                app:layout_constraintEnd_toEndOf="@id/first_actor_image_view"
+                app:layout_constraintStart_toStartOf="@id/first_actor_image_view"
+                app:layout_constraintTop_toBottomOf="@id/first_actor_image_view"/>
+            <TextView
+                android:layout_width="0dp"
+                android:layout_height="0dp"
```

Comment:
android:layout_height="0dp"
app:layout_constraintBottom_toBottomOf="@id/info_One"

u copied error

use wrap_content for android:layout_height
and remove
app:layout_constraintBottom_toBottomOf="@id/info_One"
for 2,3,4 actor's names